<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_in_screen</class>
 <widget class="QMainWindow" name="sign_in_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_bg">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_bg{
background-image: url(:/backgroud/bg_login_v1.png);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="maximumSize">
             <size>
              <width>181</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>27</number>
             </property>
             <item>
              <widget class="QPushButton" name="bt_back">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>56</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>151</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#bt_back{
	border-radius: 25px;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 64, 231, 255), stop:1 rgba(15, 181, 253, 255));
	color: rgb(255, 255, 255);
	margin-left : 30px;	
	margin-right : 30px;	
}

#bt_back:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 181, 253, 255), stop:1 rgba(89, 64, 231, 255));
}</string>
               </property>
               <property name="text">
                <string>&lt; Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="minimumSize">
             <size>
              <width>606</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>150</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_error">
               <property name="minimumSize">
                <size>
                 <width>500</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#frame_error{
	border-radius : 25px;
	background-color: rgba(255, 255, 255, 153);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QLabel" name="text_error">
                  <property name="font">
                   <font>
                    <family>Nunito</family>
                    <pointsize>19</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Vui lòng kiểm tra Email đã nhập để nhận 4 chữ số CODE và điền vào ô Mã CODE</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_x">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Nunito</family>
                    <pointsize>20</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#bt_x{
	
	background-color: rgba(255, 255, 255, 0);
}

#bt_x:hover{
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 30);
}</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="bg_login">
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>650</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#bg_login{background-color: rgba(0, 0, 0, 71);
border-radius: 20px;}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/logo/logo_only_white.png);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dang_nhap">
               <property name="maximumSize">
                <size>
                 <width>190000</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>32</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>ĐẶT LẠI MẬT KHẨU</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="email_set">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>18</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	margin-left : 20px;	
	margin-right : 20px;	
	background-color: rgba(255, 255, 255, 217);
	padding-left : 15px;
	border-radius: 30px;
	border: 2px solid rgb(45,45,45);
}

QLineEdit:focus{
	border: 2px solid rgb(90,90,90);
}

QLineEdit:hover{
	border: 2px solid rgb(90,90,90);
}
</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Email đăng ký</string>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password_set">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>18</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	margin-left : 20px;	
	margin-right : 20px;	
	background-color: rgba(255, 255, 255, 217);
	padding-left : 15px;
	border-radius: 30px;
	border: 2px solid rgb(45,45,45);
}

QLineEdit:focus{
	border: 2px solid rgb(90,90,90);
}

QLineEdit:hover{
	border: 2px solid rgb(90,90,90);
}
</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Mật khẩu mới</string>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password_set_1">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>18</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	margin-left : 20px;	
	margin-right : 20px;	
	background-color: rgba(255, 255, 255, 217);
	padding-left : 15px;
	border-radius: 30px;
	border: 2px solid rgb(45,45,45);
}

QLineEdit:focus{
	border: 2px solid rgb(90,90,90);
}

QLineEdit:hover{
	border: 2px solid rgb(90,90,90);
}
</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Xác nhận lại</string>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="code_set">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>18</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	margin-left : 20px;	
	margin-right : 20px;	
	background-color: rgba(255, 255, 255, 217);
	padding-left : 15px;
	border-radius: 30px;
	border: 2px solid rgb(45,45,45);
}

QLineEdit:focus{
	border: 2px solid rgb(90,90,90);
}

QLineEdit:hover{
	border: 2px solid rgb(90,90,90);
}
</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Mã CODE</string>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_set_new_password">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Nunito</family>
                 <pointsize>21</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#bt_set_new_password{
	border-radius: 30px;
    padding:8px 25px 7px 25px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 64, 231, 255), stop:1 rgba(15, 181, 253, 255));
	color: rgb(255, 255, 255);
	margin-left : 30px;	
	margin-right : 30px;	
}

#bt_set_new_password:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 181, 253, 255), stop:1 rgba(89, 64, 231, 255));
}</string>
               </property>
               <property name="text">
                <string>LẤY LẠI MẬT KHẨU</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Nunito</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>© FastAz.vn ® - All Rights Reserved - Dev and design by @huy8208 @QuangBlue</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/file_img.qrc"/>
 </resources>
 <connections/>
</ui>
